src/entity_gui.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of FreeRCT.
+ * FreeRCT is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * FreeRCT is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with FreeRCT. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file entity_gui.cpp Entity windows. */
+
+#include "entity_gui.h"
+
+/** Widget parts of the #EntityRemoveWindow. */
+static const WidgetPart _entity_remove_widgets[] = {
+	Intermediate(0, 1),
+		Intermediate(1, 0),
+			Widget(WT_TITLEBAR, INVALID_WIDGET_INDEX, COL_RANGE_RED), SetData(GUI_ENTITY_REMOVE_CAPTION, GUI_TITLEBAR_TIP),
+			Widget(WT_CLOSEBOX, INVALID_WIDGET_INDEX, COL_RANGE_RED),
+		EndContainer(),
+		Widget(WT_PANEL, INVALID_WIDGET_INDEX, COL_RANGE_RED),
+			Intermediate(2,0),
+				Widget(WT_CENTERED_TEXT, ERW_MESSAGE, COL_RANGE_RED),
+						SetData(GUI_ENTITY_REMOVE_MESSAGE, STR_NULL), SetPadding(5, 5, 5, 5),
+			EndContainer(),
```

Comment:
This looks wrongly indented, although it's a bit tricky; this ends the first row of line 20. Line 24 starts the 2nd row, so it should be under 'Widget' at line 21, I think.